source/usecases/usecase4/projections_B.h5.txt

```diff
@@ -4,150 +4,157 @@ GROUP "/" {
                DATA {
-                  -760.32, -953.792, -1086.96, 110.82
+                  -192.9, 142.951, 106.89, 15.1928
                }
             }
             DATASET "afferent_surface_z" {
-               DATATYPE  H5T_IEEE_F64LE
+               DATATYPE  H5T_IEEE_F32LE
                DATASPACE  SIMPLE { ( 4 ) / ( 4 ) }
                DATA {
-                  967.277, 942.154, 940.33, 274.658
+                  -781.974, 985.505, 973.581, 949.529
                }
             }
             DATASET "conductance" {
-               DATATYPE  H5T_IEEE_F64LE
+               DATATYPE  H5T_IEEE_F32LE
                DATASPACE  SIMPLE { ( 4 ) / ( 4 ) }
                DATA {
-                  0.134, 0.137783, 0.353443, 0.479719
+                  0.37601, 0.379849, 0.231088, 0.402711
                }
             }
             DATASET "conductance_scale_factor" {
-               DATATYPE  H5T_IEEE_F64LE
+               DATATYPE  H5T_IEEE_F32LE
                DATASPACE  SIMPLE { ( 4 ) / ( 4 ) }
                DATA {
-                  0.590132, 0.129219, 0.622137, 0.79398
+                  0.481629, 0.332816, 0.864134, 0.129974
                }
             }
             DATASET "decay_time" {
-               DATATYPE  H5T_IEEE_F64LE
+               DATATYPE  H5T_IEEE_F32LE
                DATASPACE  SIMPLE { ( 4 ) / ( 4 ) }
                DATA {
-                  1.96837, 1.72369, 1.86385, 1.87112
+                  1.81803, 1.62001, 1.58027, 1.8982
                }
             }
             DATASET "delay" {
-               DATATYPE  H5T_IEEE_F64LE
+               DATATYPE  H5T_IEEE_F32LE
                DATASPACE  SIMPLE { ( 4 ) / ( 4 ) }
                DATA {
-                  3.10068, 8.31519, 8.07023, 6.40024
+                  6.8027, 6.09178, 0.98478, 0.920276
                }
             }
             DATASET "depression_time" {
-               DATATYPE  H5T_IEEE_F64LE
+               DATATYPE  H5T_IEEE_F32LE
                DATASPACE  SIMPLE { ( 4 ) / ( 4 ) }
                DATA {
-                  630.699, 611.977, 644.388, 639.177
+                  695.917, 645.814, 659.098, 685.772
                }
             }
-            DATASET "facilitation_time" {
-               DATATYPE  H5T_IEEE_F64LE
+            DATASET "efferent_section_type" {
+               DATATYPE  H5T_STD_U32LE
                DATASPACE  SIMPLE { ( 4 ) / ( 4 ) }
                DATA {
-                  15.637, 21.9072, 15.7255, 18.6054
+                  1, 1, 1, 1
                }
             }
-            DATASET "n_rrp_vesicles" {
-               DATATYPE  H5T_STD_I64LE
+            DATASET "facilitation_time" {
+               DATATYPE  H5T_IEEE_F32LE
                DATASPACE  SIMPLE { ( 4 ) / ( 4 ) }
                DATA {
-                  3, 1, 3, 1
+                  14.1445, 24.0375, 16.515, 21.4153
                }
             }
-            DATASET "spine_length" {
```

Comment:
does that mean that spine_length disappeared ? I don't find it at first glance

Reply:
I just looked into this. `spine_length` is not a field for projections. The reason for that, is that the definition of `spine_length` is "Distance between the two surface positions in µm". Since virtual nodes do not have surface positions, this cannot be computed.

Reply:
indeed. I forgot we had a special type for projections edges. I was expecting a regular chemical synapse